packages/kit/package.json

```
@@ -23,7 +23,7 @@
 		"set-cookie-parser": "^2.5.1",
 		"sirv": "^2.0.2",
 		"tiny-glob": "^0.2.9",
-		"undici": "5.20.0"
+		"undici": "^5.22.0"
```

AFAIK we wanted to pin this mostly since undici is still somewhat unstable - I propose to use `~` instead
```suggestion
		"undici": "~5.22.0"
```